src/pages/RestaurantPage/RestaurantPage.js

```diff
@@ -2,36 +2,22 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components/macro'
 import Card from './../../components/Card/Card'
 import Header from './../../components/Header/Header'
-import locations from './../../locations.json'
 
 RestaurantPage.propTypes = {
   onNavigate: PropTypes.func.isRequired,
 }
 
-export default function RestaurantPage() {
+export default function RestaurantPage({ handleBookmark, restaurants }) {
   return (
     <Wrapper>
       <Header>Food and Drinks</Header>
       <ListWrapper>
-        {locations.restaurants.map(restaurant => {
-          const {
-            id,
-            image,
-            title,
-            text,
-            address,
-            openingHours,
-            entry,
-          } = restaurant
+        {restaurants.map(restaurants => {
```

Comment:
Probably restructure here aswell if possible (not sure if it works with `...restaurants` as intended)